Dockerfile

```
@@ -0,0 +1,7 @@
+FROM stoplight/prism:latest
+
+COPY /docs /usr/src/prism/packages/cli/docs/
+
+EXPOSE 80
+
+CMD ["mock", "-h", "0.0.0.0", "-p", "80", "docs/openapi/openapi.yml"]
```

Not sure if it should be the path from the Docker Image? 🤔 

```suggestion
CMD ["mock", "-h", "0.0.0.0", "-p", "80", "/usr/src/prism/packages/cli/docs/openapi/openapi.yml"]
```

---

Yes, but as we copied to that path ([this line](https://github.com/nimblehq/api-mock-templates/pull/3/files/9cb6f6d8d270608c88b92c91ba47813a9f28d016#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R3)). So I think it's good to use the short path instead

---

Ahh my bad. Thanks for your explanation.